src/Analyzer.Core/TemplateAnalyzer.cs

```diff
@@ -5,13 +5,15 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using Microsoft.Azure.Templates.Analyzer.BicepProcessor;
```

Comment:
Can you double-check the added using statements are necessary?  This one doesn't seem like it.